src/pages/email-change-request/[token].tsx

```diff
@@ -27,12 +89,65 @@ const EmailChangeRequest: React.FunctionComponent = () => {
               Email Change Request
             </h2>
             <div className="sm:mt-8 mt-4 sm:mx-auto sm:w-full sm:max-w-xl">
-              <p className="text-center pb-8">
+              <p className="text-center pb-4">
                 Click 'confirm' to complete your email change request.
               </p>
+              {validToken && (
+                <p className="text-center mb-4 p-4 bg-blue-50 dark:bg-gray-800 rounded">
+                  {currentEmail} → {newEmail}
+                </p>
+              )}
+              {!validToken && (
+                <div
+                  className="relative px-4 py-3 mb-4 leading-normal text-red-700 bg-red-100 dark:text-red-100 dark:bg-red-800 rounded"
+                  role="alert"
+                >
+                  <span className="absolute inset-y-0 left-0 flex items-center ml-4">
+                    <svg
+                      className="w-6 h-6"
+                      xmlns="http://www.w3.org/2000/svg"
+                      fill="none"
+                      viewBox="0 0 24 24"
+                      stroke="currentColor"
+                    >
+                      <path
+                        strokeLinecap="round"
+                        strokeLinejoin="round"
+                        strokeWidth="2"
+                        d="M10 14l2-2m0 0l2-2m-2 2l-2-2m2 2l2 2m7-2a9 9 0 11-18 0 9 9 0 0118 0z"
+                      />
+                    </svg>
+                  </span>
+                  <p className="ml-8">
+                    This link for changing your email has been used or has
+                    expired. Feel free to{' '}
+                    <a
+                      className="font-bold underline transition-colors ease-in-out duration-150"
+                      href="/user"
+                    >
+                      request a new link
+                    </a>
+                    . Please contact{' '}
+                    <a
+                      className="font-bold underline transition-colors ease-in-out duration-150"
+                      href="mailto:support@egghead.io"
+                    >
+                      support@egghead.io
+                    </a>{' '}
+                    to get further help with this request.
+                  </p>
+                </div>
+              )}
               <div className="flex justify-center items-center w-full">
                 <button
-                  className="text-white bg-red-500 border-0 py-2 px-8 focus:outline-none hover:bg-red-600 rounded"
+                  className={cx(
```

Comment:
I'm hesitant to bring in a new library for this single use case. We are generally just using template strings for this circumstance. Can you explain why this is needed?

I think template approach is less likely to break Tailwind.

Regardless, need to import its type definitions or add it as a global module if it's essential.

Reply:
It's not essential, I'll switch this over to template strings. I was kinda surprised that classnames wasn't already being used. I'm not aware of any impact it would have on Tailwind.